### PR TITLE
Readme fix

### LINKED ISSUE #321: Fix README.md files
## 🐛 Bug Report

Many readme files contain the following issues.
- Broken or missing links
- Grammatical errors
- Incorrect ordering of tags

### Have you read the [Contributing Guidelines](https://github.com/Kushal997-das/Project-Guidance/blob/main/CONTRIBUTING.md) ?

Yes

### PR DESCRIPTION
**Pull request by rajdip-b**
### Have you read the [Contributing Guidelines](https://github.com/Kushal997-das/Project-Guidance/blob/main/CONTRIBUTING.md) ?

Yes

### Description

Fixed all of the issues with broken/missing links 

Fixes #321 

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)

### CODE REVIEW

File: Machine Learning and Data Science/README.md [hunk 1]
```diff
@@ -1 +1,57 @@
...
+
+  - [ Prediction using Decision Tree Algorithm ](https://github.com/Kushal997-das/Project-Guidance/tree/main/Machine%20Learning%20and%20Data%20Science/Intermediate/Prediction%20using%20Decision%20Tree%20Algorithm)<br>
+
+  - [ Explore Buisness Analytics ](https://github.com/Kushal997-das/Project-Guidance/tree/main/Machine%20Learning%20and%20Data%20Science/Intermediate/To%20Explore%20Buisness%20Analytics)<br>
+
+    <br>
+
+- <h1>Advanced 🚀 </h1>
+
+  s- [ Detox ](https://github.com/Kushal997-das/Project-Guidance/tree/main/Machine%20Learning%20and%20Data%20Science/Advanced/Detox)<br>
```
Comment (Kushal997-das) [inline Machine Learning and Data Science/README.md:35]: Add `-` instead `s`.

File: Desktop Application/README.md [hunk 2]
```diff
@@ -54,28 +66,52 @@
...
     
     - [<h3 >  Track Phone Number Location </h3>](https://github.com/Kushal997-das/Project-Guidance/tree/main/Desktop%20Application/Basic/Python/Track%20Phone%20number%20location)
+
+    - [<h3 >  Vaccination Centre Locator  </h3>](https://github.com/Kushal997-das/Project-Guidance/tree/main/Desktop%20Application/Basic/Python/Vaccination%20Centre%20Locator)
+
+    - [<h3 >  Vector Calculator  </h3>](https://github.com/Kushal997-das/Project-Guidance/tree/main/Desktop%20Application/Basic/Python/Vector%20Calculator)
         
-    - [<h3 >  Story Generator  </h3>](https://github.com/Kushal997-das/Project-Guidance/tree/main/Desktop%20Application/Basic/Python/Story-Generator)
+    </br>
+  - [<h2> SQL </h2>](https://github.com/Kushal997-das/Project-Guidance/tree/main/Desktop%20Application/Basic/SQL)
```
Comment (Kushal997-das) [inline Desktop Application/README.md:83]: This link is not working !

**Review by Kushal997-das — CHANGES_REQUESTED** [review]
@rajdip-b Make the required changes 👍

File: Desktop Application/README.md (hunk 2 above)
Comment (Kushal997-das) [reply Desktop Application/README.md:83]: This SQL link

File: Machine Learning and Data Science/README.md (hunk 1 above)
Comment (rajdip-b) [reply Machine Learning and Data Science/README.md]: fixed

File: Desktop Application/README.md (hunk 2 above)
Comment (rajdip-b) [reply Desktop Application/README.md:83]: Yes that's an issue i want to raise. I tried to fix it so many times but it doesn't fix IDK why. The link is not broken

File: Machine Learning and Data Science/README.md (hunk 1 above)
Comment (Kushal997-das) [reply Machine Learning and Data Science/README.md]: Okey

File: Desktop Application/README.md (hunk 2 above)
Comment (Kushal997-das) [reply Desktop Application/README.md:84]: Okey @rajdip-b I'll try to fix it by my end and yes great work 💯

Comment (rajdip-b) [reply Desktop Application/README.md:84]: Great! Thanks <3

**Pull request merged**